bc_obps/registration/fixtures/mock/operator.json

```diff
@@ -255,8 +255,8 @@
       "contacts": [1],
       "status": "Draft",
       "is_new": true,
-      "verified_at": "2024-01-03 08:09:51.133556-08",
-      "verified_by": "00000000-0000-0000-0000-000000000002"
+      "verified_at": null,
+      "verified_by": null
```

Comment:
Out of curiosity, why set this to `null` rather than remove it?

Reply:
@marcellmueller capturing that we will defer to @Sepehr-Sobhani or @BCerki for answers to the api changes

Reply:
If it was me who put it in, I just didn't think it through, no reason